components/Splash/index.tsx

```diff
@@ -1,35 +1,46 @@
 import Link from 'next/link';
 import React, { useContext } from 'react';
 import { AppContext } from '../../pages/_app';
+import colors from '../../styles/_variables.module.scss';
 import styles from '../../styles/Splash.module.scss';
-import Stars from './Stars';
-import SunMoon from './SunMoon';
+import { ILink } from '../../utils';
 
 interface SplashProps {
-  mousePos: number[];
+  heading: string;
+  description: string;
+  children: JSX.Element;
+  buttons?: ILink[];
+  halve?: boolean;
 }
 
 function Splash(props: SplashProps): JSX.Element {
+  const {heading, description, children, buttons, halve} = props;
```

Comment:
nit: why not just stick the destructuring inside the function signature like
```typescript
function Splash({heading, description, children, buttons, halve}: SplashProps): JSX.Element
```
not sure if that's legal TS or not tho